diff --git a/prio_test2/PRIO_test2/__pycache__/__init__.cpython-310.pyc b/prio_test2/PRIO_test2/__pycache__/__init__.cpython-310.pyc
index a2045a8..9746d41 100644
Binary files a/prio_test2/PRIO_test2/__pycache__/__init__.cpython-310.pyc and b/prio_test2/PRIO_test2/__pycache__/__init__.cpython-310.pyc differ
diff --git a/prio_test2/PRIO_test2/__pycache__/settings.cpython-310.pyc b/prio_test2/PRIO_test2/__pycache__/settings.cpython-310.pyc
index 3ec38f9..4be19b3 100644
Binary files a/prio_test2/PRIO_test2/__pycache__/settings.cpython-310.pyc and b/prio_test2/PRIO_test2/__pycache__/settings.cpython-310.pyc differ
diff --git a/prio_test2/PRIO_test2/__pycache__/urls.cpython-310.pyc b/prio_test2/PRIO_test2/__pycache__/urls.cpython-310.pyc
index 7c7fd36..b24fa79 100644
Binary files a/prio_test2/PRIO_test2/__pycache__/urls.cpython-310.pyc and b/prio_test2/PRIO_test2/__pycache__/urls.cpython-310.pyc differ
diff --git a/prio_test2/PRIO_test2/__pycache__/wsgi.cpython-310.pyc b/prio_test2/PRIO_test2/__pycache__/wsgi.cpython-310.pyc
index 4809c85..727d8ec 100644
Binary files a/prio_test2/PRIO_test2/__pycache__/wsgi.cpython-310.pyc and b/prio_test2/PRIO_test2/__pycache__/wsgi.cpython-310.pyc differ
diff --git a/prio_test2/PRIO_test2/settings.py b/prio_test2/PRIO_test2/settings.py
index 2b996cc..acccd0f 100644
--- a/prio_test2/PRIO_test2/settings.py
+++ b/prio_test2/PRIO_test2/settings.py
@@ -39,6 +39,7 @@ INSTALLED_APPS = [
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'betterforms',
 ]
 
 MIDDLEWARE = [
@@ -129,9 +130,9 @@ STATIC_URL = "static/"
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-# STATICFILES_DIRS = [r"/home/aforth/prio_test2/hello/static"]
+STATICFILES_DIRS = [r"/home/aforth/Pr.io/prio_test2/hello/static"]
 
-STATICFILES_DIRS = [r"C:\Users\Lenovo x240\PRIO_test2\hello\static"]
+#STATICFILES_DIRS = [r"C:\Users\Lenovo x240\PRIO_test2\hello\static"]
 
 ## insert file path here
 
diff --git a/prio_test2/db.sqlite3 b/prio_test2/db.sqlite3
index aabdbed..74c5df6 100644
Binary files a/prio_test2/db.sqlite3 and b/prio_test2/db.sqlite3 differ
diff --git a/prio_test2/hello/__pycache__/__init__.cpython-310.pyc b/prio_test2/hello/__pycache__/__init__.cpython-310.pyc
index 9deded4..910146a 100644
Binary files a/prio_test2/hello/__pycache__/__init__.cpython-310.pyc and b/prio_test2/hello/__pycache__/__init__.cpython-310.pyc differ
diff --git a/prio_test2/hello/__pycache__/admin.cpython-310.pyc b/prio_test2/hello/__pycache__/admin.cpython-310.pyc
index b50079e..4e95a71 100644
Binary files a/prio_test2/hello/__pycache__/admin.cpython-310.pyc and b/prio_test2/hello/__pycache__/admin.cpython-310.pyc differ
diff --git a/prio_test2/hello/__pycache__/apps.cpython-310.pyc b/prio_test2/hello/__pycache__/apps.cpython-310.pyc
index f6ca3aa..a831f98 100644
Binary files a/prio_test2/hello/__pycache__/apps.cpython-310.pyc and b/prio_test2/hello/__pycache__/apps.cpython-310.pyc differ
diff --git a/prio_test2/hello/__pycache__/forms.cpython-310.pyc b/prio_test2/hello/__pycache__/forms.cpython-310.pyc
index 69cafe0..807d428 100644
Binary files a/prio_test2/hello/__pycache__/forms.cpython-310.pyc and b/prio_test2/hello/__pycache__/forms.cpython-310.pyc differ
diff --git a/prio_test2/hello/__pycache__/models.cpython-310.pyc b/prio_test2/hello/__pycache__/models.cpython-310.pyc
index c81706c..c742a9d 100644
Binary files a/prio_test2/hello/__pycache__/models.cpython-310.pyc and b/prio_test2/hello/__pycache__/models.cpython-310.pyc differ
diff --git a/prio_test2/hello/__pycache__/urls.cpython-310.pyc b/prio_test2/hello/__pycache__/urls.cpython-310.pyc
index 88f60a0..11ec9ae 100644
Binary files a/prio_test2/hello/__pycache__/urls.cpython-310.pyc and b/prio_test2/hello/__pycache__/urls.cpython-310.pyc differ
diff --git a/prio_test2/hello/__pycache__/views.cpython-310.pyc b/prio_test2/hello/__pycache__/views.cpython-310.pyc
index 43e83c0..096741e 100644
Binary files a/prio_test2/hello/__pycache__/views.cpython-310.pyc and b/prio_test2/hello/__pycache__/views.cpython-310.pyc differ
diff --git a/prio_test2/hello/admin.py b/prio_test2/hello/admin.py
index 6b55408..0150a83 100644
--- a/prio_test2/hello/admin.py
+++ b/prio_test2/hello/admin.py
@@ -5,3 +5,4 @@ from .models import Task,Subject
 
 admin.site.register(Task)
 admin.site.register(Subject)
+
diff --git a/prio_test2/hello/forms.py b/prio_test2/hello/forms.py
index dcd63d4..d88d7d0 100644
--- a/prio_test2/hello/forms.py
+++ b/prio_test2/hello/forms.py
@@ -1,7 +1,18 @@
 from django import forms
 from .models import Subject
+from betterforms.multiform import MultiModelForm
+
 
 class Subjectform(forms.ModelForm):
     class Meta:
         model = Subject
-        fields = ['subName', 'numUnits','subStart','subEnd'] 
\ No newline at end of file
+        fields = ['subName', 'numUnits','subStart','subEnd','reqName1','gradeNum1'
+                  ,'reqName2','gradeNum2',
+                  'reqName3','gradeNum3',
+                  'reqName4','gradeNum4',
+                  'reqName5','gradeNum5'] 
+
+# class Requirementform(forms.ModelForm):
+#     class Meta:
+#         model = Requirement
+#         fields = ['reqName', 'subName','gradeNum'] 
diff --git a/prio_test2/hello/migrations/__pycache__/0001_initial.cpython-310.pyc b/prio_test2/hello/migrations/__pycache__/0001_initial.cpython-310.pyc
index b899d73..55d22a3 100644
Binary files a/prio_test2/hello/migrations/__pycache__/0001_initial.cpython-310.pyc and b/prio_test2/hello/migrations/__pycache__/0001_initial.cpython-310.pyc differ
diff --git a/prio_test2/hello/migrations/__pycache__/0002_subject.cpython-310.pyc b/prio_test2/hello/migrations/__pycache__/0002_subject.cpython-310.pyc
index 84876cd..d872845 100644
Binary files a/prio_test2/hello/migrations/__pycache__/0002_subject.cpython-310.pyc and b/prio_test2/hello/migrations/__pycache__/0002_subject.cpython-310.pyc differ
diff --git a/prio_test2/hello/migrations/__pycache__/__init__.cpython-310.pyc b/prio_test2/hello/migrations/__pycache__/__init__.cpython-310.pyc
index bc7fd1c..1c0e8e3 100644
Binary files a/prio_test2/hello/migrations/__pycache__/__init__.cpython-310.pyc and b/prio_test2/hello/migrations/__pycache__/__init__.cpython-310.pyc differ
diff --git a/prio_test2/hello/models.py b/prio_test2/hello/models.py
index a9fad47..8cfaad4 100644
--- a/prio_test2/hello/models.py
+++ b/prio_test2/hello/models.py
@@ -2,10 +2,10 @@ from django.db import models
 
 # Create your models here.
 
-
+# Each task is my requirement type
 class Task(models.Model):
-    reqID = models.IntegerField()
-    reqName = models.CharField(max_length=50)
+    taskName = models.CharField(max_length=50,default = "")
+    # reqName = models.ForeignKey("hello.Requirement",on_delete=models.CASCADE)
     dueDate = models.DateTimeField()
 
 
@@ -15,6 +15,29 @@ class Subject(models.Model):
     numUnits = models.IntegerField()
     subStart = models.DateTimeField()
     subEnd = models.DateTimeField()
+    reqName1 = models.CharField(max_length=50)
+    gradeNum1 = models.DecimalField(max_digits=5,decimal_places=2)
+
+    reqName2 = models.CharField(max_length=50)
+    gradeNum2 = models.DecimalField(max_digits=5,decimal_places=2)
+
+    reqName3 = models.CharField(max_length=50)
+    gradeNum3 = models.DecimalField(max_digits=5,decimal_places=2)
+
+    reqName4 = models.CharField(max_length=50)
+    gradeNum4 = models.DecimalField(max_digits=5,decimal_places=2)
+
+    reqName5 = models.CharField(max_length=50)
+    gradeNum5 = models.DecimalField(max_digits=5,decimal_places=2)
 
     def __str__(self):
-        return self.subName + '   ' +  str(self.numUnits) + '   ' + str(self.subStart.day) +'   '+ str(self.subEnd.day)
\ No newline at end of file
+        return self.subName
+
+#Model ng requirements is para sa grading system talaga
+# class Requirement(models.Model):
+#     reqName = models.CharField(max_length=50)
+#     subName = models.ForeignKey("hello.Subject",on_delete=models.CASCADE)
+#     gradeNum = models.DecimalField(max_digits=5,decimal_places=2)
+
+#     def __str__(self):
+#         return self.reqName
\ No newline at end of file
diff --git a/prio_test2/hello/static/html/add_subject.html b/prio_test2/hello/static/html/add_subject.html
index 05ff873..323a940 100644
--- a/prio_test2/hello/static/html/add_subject.html
+++ b/prio_test2/hello/static/html/add_subject.html
@@ -2,7 +2,7 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <!-- <script type="text/javascript" src="{% static 'js/add_subject.js' %}" defer></script> -->
+        <!-- <script type="text/javascript" src="{% static 'js/add_subject.js' %}" defer></script>  -->
         <title>Add Subject Page</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" type="text/css" href="{% static 'css/main.css' %}">
@@ -46,7 +46,24 @@
                         <!-- {{ form.subName }} -->
                         <h3>Grading System / Breakdown</h3>
                         <div id="gradeSysBox">
-                            <button id="add_gradeSys" class="control-button" onclick="add_gradeSys()">+</button>
+
+                            <input id="gradeSys-Type-1" class="gradeSys-Type input-field" type="text" placeholder="e.g. Exam">
+                            <input id="gradeSys-Points-1" class="gradeSys-Points input-field" type="number" placeholder="%, e.g. 30" min="0" max="100">
+
+                            <input id="gradeSys-Type-2" class="gradeSys-Type input-field" type="text" placeholder="e.g. Exam">
+                            <input id="gradeSys-Points-2" class="gradeSys-Points input-field" type="number" placeholder="%, e.g. 30" min="0" max="100">
+
+                            <input id="gradeSys-Type-3" class="gradeSys-Type input-field" type="text" placeholder="e.g. Exam">
+                            <input id="gradeSys-Points-3" class="gradeSys-Points input-field" type="number" placeholder="%, e.g. 30" min="0" max="100">
+
+                            <input id="gradeSys-Type-4" class="gradeSys-Type input-field" type="text" placeholder="e.g. Exam">
+                            <input id="gradeSys-Points-4" class="gradeSys-Points input-field" type="number" placeholder="%, e.g. 30" min="0" max="100">
+
+                            <input id="gradeSys-Type-5" class="gradeSys-Type input-field" type="text" placeholder="e.g. Exam">
+                            <input id="gradeSys-Points-5" class="gradeSys-Points input-field" type="number" placeholder="%, e.g. 30" min="0" max="100">
+
+
+                            <!-- <button id="add_gradeSys" class="control-button" onclick="add_gradeSys()">+</button> -->
                         </div><br><br>
                         
                         <div id="startEnd">
diff --git a/prio_test2/hello/static/html/edit_subject.html b/prio_test2/hello/static/html/edit_subject.html
index b11de65..d3b479e 100644
--- a/prio_test2/hello/static/html/edit_subject.html
+++ b/prio_test2/hello/static/html/edit_subject.html
@@ -35,7 +35,7 @@
               {% if subjects %}
                   <ul>
                   {% for subject in subjects %}
-                      <li><a href="{% url 'subject_details' subject.id %}">{{ subject.subName }}</a></li>
+                      <li><a href="{% url 'subject_details' subject.id %}">{{ subject}}</a></li>
                   {% endfor %}
                   </ul>
               {% else %}
diff --git a/prio_test2/hello/views.py b/prio_test2/hello/views.py
index c3c1332..db91538 100644
--- a/prio_test2/hello/views.py
+++ b/prio_test2/hello/views.py
@@ -26,16 +26,37 @@ def addSubForm(request):
     # print("Hello World????? 1", dir(request))
     # print(request.body)
     if request.method == "POST":
-        print("TSENELIN")
-        subName = request.POST["subName"]
-        numUnits = request.POST["numUnits"]
-        subStart = request.POST["subStart"]
-        subEnd = request.POST["subEnd"]
-        print(subName, numUnits, subStart, subEnd)
-        addSub_details = Subject(subName = subName, numUnits = numUnits, subStart = subStart, subEnd = subEnd)
+        
+        reqsdict = {
+        "subName" : request.POST["subName"],
+        "numUnits" : request.POST["numUnits"],
+        "subStart" : request.POST["subStart"],
+        "subEnd" : request.POST["subEnd"],
+        
+        "reqName1" : request.POST["reqName1"],
+        "gradeNum1" : request.POST["gradeNum1"],
+
+        "reqName2" : request.POST["reqName2"],
+        "gradeNum2" : request.POST["gradeNum2"],
+
+        "reqName3" : request.POST["reqName3"],
+        "gradeNum3" : request.POST["gradeNum3"],
+
+        "reqName4" : request.POST["reqName4"],
+        "gradeNum4" : request.POST["gradeNum4"],
+
+        "reqName5" : request.POST["reqName5"],
+        "gradeNum5" : request.POST["gradeNum5"],
+        }
+     
+
+
+        # print(subName, numUnits, subStart, subEnd)
+        addSub_details = Subject(**reqsdict)
         addSub_details.save()
+
+       
     
-    print("KIMMY")
     return render(request, 'add_subject.html')
 
 def checkSub(request):
diff --git a/prio_test2/members/__pycache__/__init__.cpython-310.pyc b/prio_test2/members/__pycache__/__init__.cpython-310.pyc
index cebab2b..680b0e4 100644
Binary files a/prio_test2/members/__pycache__/__init__.cpython-310.pyc and b/prio_test2/members/__pycache__/__init__.cpython-310.pyc differ
diff --git a/prio_test2/members/__pycache__/admin.cpython-310.pyc b/prio_test2/members/__pycache__/admin.cpython-310.pyc
index 6c8a2bd..0659cf6 100644
Binary files a/prio_test2/members/__pycache__/admin.cpython-310.pyc and b/prio_test2/members/__pycache__/admin.cpython-310.pyc differ
diff --git a/prio_test2/members/__pycache__/apps.cpython-310.pyc b/prio_test2/members/__pycache__/apps.cpython-310.pyc
index 3cee69f..88d42cd 100644
Binary files a/prio_test2/members/__pycache__/apps.cpython-310.pyc and b/prio_test2/members/__pycache__/apps.cpython-310.pyc differ
diff --git a/prio_test2/members/__pycache__/models.cpython-310.pyc b/prio_test2/members/__pycache__/models.cpython-310.pyc
index 5c1bf5e..627dcf3 100644
Binary files a/prio_test2/members/__pycache__/models.cpython-310.pyc and b/prio_test2/members/__pycache__/models.cpython-310.pyc differ
diff --git a/prio_test2/members/__pycache__/urls.cpython-310.pyc b/prio_test2/members/__pycache__/urls.cpython-310.pyc
index f915f4e..0572ac9 100644
Binary files a/prio_test2/members/__pycache__/urls.cpython-310.pyc and b/prio_test2/members/__pycache__/urls.cpython-310.pyc differ
diff --git a/prio_test2/members/__pycache__/views.cpython-310.pyc b/prio_test2/members/__pycache__/views.cpython-310.pyc
index e6c8794..c6c218e 100644
Binary files a/prio_test2/members/__pycache__/views.cpython-310.pyc and b/prio_test2/members/__pycache__/views.cpython-310.pyc differ
diff --git a/prio_test2/members/migrations/__pycache__/__init__.cpython-310.pyc b/prio_test2/members/migrations/__pycache__/__init__.cpython-310.pyc
index 6a7e3f9..3e4455c 100644
Binary files a/prio_test2/members/migrations/__pycache__/__init__.cpython-310.pyc and b/prio_test2/members/migrations/__pycache__/__init__.cpython-310.pyc differ
